src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.tsx

```diff
@@ -71,6 +83,16 @@ export const TrackPanelList = () => {
           />
         </section>
       ) : null}
+
+      <section>
+        <div className={styles.sectionContent}>
```

Comment:
![image](https://user-images.githubusercontent.com/6834224/175878090-16d2cd8a-ee1c-4514-a554-034f1a41c556.png)

- This class does not exist
- What would it do if it existed?
- Is there any point in having this wrapper at all?